components/hero-home.tsx

```diff
@@ -70,17 +70,16 @@ export default function HeroHome() {
               data-aos="zoom-y-out"
               data-aos-delay={150}
             >
-              The website builder you're <br className="max-lg:hidden" />
-              looking for
+              Learning by Creating <br className="max-lg:hidden" />
```

Comment:
This is a change that should be merged yesterday. We might be trying to merge into a wrong branch